src/native/http_request_response.c

```diff
@@ -569,12 +570,24 @@ static bool s_fill_out_request(
         jbyteArray jname = (*env)->GetObjectField(env, jHeader, s_http_header.name);
         jbyteArray jvalue = (*env)->GetObjectField(env, jHeader, s_http_header.value);
 
+        size_t name_len = (*env)->GetArrayLength(env, jname);
+        jbyte *name = (*env)->GetPrimitiveArrayCritical(env, jname, NULL);
+        struct aws_byte_cursor name_cursor = aws_byte_cursor_from_array(name, name_len);
+
+        size_t value_len = (*env)->GetArrayLength(env, jvalue);
+        jbyte *value = (*env)->GetPrimitiveArrayCritical(env, jvalue, NULL);
```

Comment:
the lens and jbyte* can all be const

Reply:
the jbytes cant, because I have to hand them back to the JVM as mutable pointers...